charts/prometheus-snmp-exporter/values.yaml

```diff
@@ -127,6 +127,8 @@ podAnnotations: {}
 extraArgs: []
 #  --history.limit=1000
 
+envFrom: []
+
```

Comment:
Can you add an example configuration in a comment here to help future users of the value, something like:
```
# - secretRef:
#     name: name-of-secret
```
Similar to how you did in the description of #4438. You can see where we have done similar elsewhere in the values. Thanks!

Reply:
Of course - done. Thanks for reviewing.